### DISCUSSION TITLE
This is Really Cool! ✨️

### DISCUSSION THREAD

**Discussion by zicklag** [egraphs-good/egg#63] [General]
I'm not one who knows any of the theory around this yet, but I went through the tutorial and this seems really cool. 😃

Also, very good job on the tutorial. It was one of the most readable explanations of a potentially confusing topic I've read.

I don't know what I may get the chance to use this on, but I'm definitely going to be thinking about it. 😉

**Comment by mwillsey** [comment]
Thanks for the kind words! It's very nice to hear things like that. :smile:

**Reply by ghost** [reply]
I really have to say, after working now for a while with this crate, the API is very, very well designed and the functionality absolutely amazing. This is exactly what I have been looking for for my project.
